meson.build

```diff
@@ -111,8 +111,8 @@ endif
 if get_option('rpgp')
   # The rpgp pkg-config file is currently incorrect, so just try
   # linking it instead for now.
-  rpgp = dependency('rpgp')
-  #rpgp = cc.find_library('rpgp')
+  # rpgp = dependency('rpgp')
```

Comment:
If we need to do this could we do this in a separate PR please? I'm not sure why you need it and would like to see if there's an other solution to whatever made you do this

Reply:
removed here.